NitroxServer/GameLogic/Entities/Spawning/BatchEntitySpawner.cs

```diff
@@ -359,8 +359,51 @@ private List<Entity> ConvertComponentPrefabsToEntities(List<PrefabAsset> prefabs
                              counter++,
                              parent);
 
-                prefabEntity.ChildEntities = ConvertComponentPrefabsToEntities(prefab.Children, prefabEntity, deterministicBatchGenerator);
+                if (prefab.Name.Contains("(Placeholder)"))
```

Comment:
Maybe leave a comment for this little buddy ?
 I personnaly have no clue about it

Reply:
Sorry what about this part?  That line that is removed here is actually just altered and moved down below.